app/services/RuleProvisionerService.scala

```diff
@@ -25,27 +25,36 @@ class RuleProvisionerService(
   /**
     * Update the rules in our matcherPool, given a ruleResource.
     */
-  def updateRules(ruleResource: RuleResource, date: Date): Unit = {
+  def updateRules(ruleResource: RuleResource, date: Date): Either[List[Throwable], Unit] = {
     matcherPool.removeAllMatchers()
 
-    ruleResource.rules.groupBy(_.category).foreach {
+    addLTMatcherToPool(matcherPool, Nil, ruleResource.ltDefaultRuleIds)
```

Comment:
Is it worth trying to capture errors from the default rules? At the moment they are discarded.

Reply:
We don't pass any errors as a result of the enable op – but we probably should, for example if we're trying to enable something that doesn't exist! I'll add that logic.